src/components/GameItem.js

```diff
@@ -2,15 +2,42 @@ import React, { Component } from 'react';
 import '../App.css';
 import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
+import { spawn } from 'child_process';
+const randomObjProp = require('random-obj-prop');
+
 
 class GameItem extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      classForX: "game-item",
+      alreadyScored: false,
+    }
+  }
+
   static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
   onItemClicked = () => {
     // Fill this in for Wave 2!
+   if (this.props.type === "litter"){
+      this.setState({
+      classForX: "game-item spotted-litter"
+      })
+      if (this.state.alreadyScored === false){
+        this.props.onClickCallback()
+        this.setState({
+          alreadyScored: true
```

Comment:
Calling setState twice slows down the app because it needs to re-render twice and isn't necessary because classForX only needs to be updated if it hasn't been scored yet.
To further simplify, classForX could just been removed and the class could be determined in the render function based on the type and the alreadyScored state.